api/v2.0/swagger.yaml

```diff
@@ -9016,3 +9144,89 @@ definitions:
         type: string
         format: date-time
         description: The creation time of the accessory
+
+  ScanDataExportRequest:
+    type: object
+    description: The criteria to select the scan data to export.
+    properties:
+      job_name:
```

Comment:
is it necessary? User might not know it.

Reply:
This is a pure UI requirement that helps users to distinguish between exporting jobs